pipx/util.py

```diff
@@ -94,11 +94,12 @@ def get_site_packages(python: Path) -> Path:
 def run(cmd: Sequence[Union[str, Path]], check=True) -> int:
     """Run arbitrary command as subprocess"""
 
+    env = {k:v for k,v in os.environ.items() if k.upper() != 'PYTHONPATH'}
```

Comment:
@cs01  Should we add some changelog for this?

Reply:
Yes

Reply:
done https://github.com/pipxproject/pipx/pull/252/commits/eed7ff87f9ef4ff772ca66c42f899c1df1b26ec4